client/java/src/main/java/io/openlineage/client/Clients.java

```diff
@@ -13,6 +14,10 @@ public static OpenLineageClient newClient() {
   }
 
   public static OpenLineageClient newClient(ConfigPathProvider configPathProvider) {
+    String isDisabled = Environment.getEnvironmentVariable("OPENLINEAGE_DISABLED");
+    if (isDisabled != null && (isDisabled.equals("true") || isDisabled.equals("True"))) {
```

Comment:
consider using `equalsIgnoreCase`

Reply:
Use `Boolean.parseBoolean` - it does the null check and the case-insensitive comparison

Reply:
Thanks - `Boolean.parseBoolean` is exactly what I need.